requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests-toolbelt==0.8.0
-contentful==1.11.4
+contentful==1.7.0
```

Comment:
@Piotr-Rogulski why 1.11.4 => 1.7.0?

Reply:
@eshlox My bad... Wrongly resolved a conflict. (support for envs are >= 1.7.0)